packages/api/core/spec/fixture/custom_init/index.js

```diff
@@ -4,7 +4,7 @@ const baseTemplate = require('@electron-forge/template-base').default;
 const fs = require('fs-extra');
 
 module.exports = {
-  requiredForgeVersion: '>= 6.0.0-beta.1',
+  requiredForgeVersion: '>= 8.0.0-alpha.0',
```

Comment:
Note: this started failing because `semver.satisfies` doesn't accept prerelease tags from different major versions as a design choice:

https://docs.npmjs.com/cli/v6/using-npm/semver#prerelease-tags